### PR TITLE
Adding help to the tracking

### LINKED ISSUE #11518: Add description to the UI what the Inventory Tracked flag is about
**Describe the proposed solution**

For better UX for the user there should be a description what the `Tracked` flag in the Product Variant Inventory is for and what effect it has. Me personally it always confuse it if it need to be activated or disabled for which case 🙈 .

### PR DESCRIPTION
**Pull request by mamazu**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #11518
| License         | MIT

Adding a little help text to the tracked flag of Sylius inventory.
screenshot

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Product/Tab/_inventory.html.twig [hunk 1]
```diff
@@ -3,7 +3,11 @@
     <div class="ui attached segment">
         {% if product.simple %}
             {{ form_row(form.variant.onHand) }}
+            <br />
+
             {{ form_row(form.variant.tracked) }}
+            <div style="color: rgba(0, 0, 0, .6)">{{ form_help(form.variant.tracked) }}</div>
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Product/Tab/_inventory.html.twig:9]: IMHO inline styles should be discouraged. Can you add a new style to src/Sylius/Bundle/UiBundle/Resources/private/sass/main.scss?

**Comment by mamazu** [comment]
This is how it looks now. The field is a little to small for the flag though but it is consistent with the semantic ui styles.
screenshot_2

File: src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Tab/_inventory.html.twig [hunk 2]
```diff
@@ -2,7 +2,10 @@
     <h3 class="ui dividing header">{{ 'sylius.ui.inventory'|trans }}</h3>
     <div class="ui segment">
         {{ form_row(form.onHand) }}
+
         {{ form_row(form.tracked) }}
+        <div style="color: rgba(0, 0, 0, .6)">{{ form_help(form.tracked) }}</div>
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Tab/_inventory.html.twig:7]: can we remove inline styles here as well?

Comment (mamazu) [reply src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Tab/_inventory.html.twig]: Yes.

**Comment by lchrusciel** [comment]
Thank you, @mamazu! :tada:

**Comment by alexander-schranz** [comment]
@mamazu Thank you 👍

**Pull request merged**